_maps/map_files/Pahrump/Pahrump.dmm

```diff
@@ -220,8 +233,14 @@
 "aey" = (/turf/closed/wall/r_wall/rust,/area/f13/tunnel)
 "aez" = (/obj/effect/decal/marking{tag = "icon-doublevertical"; icon_state = "doublevertical"},/turf/open/indestructible/ground/outside/road{icon_state = "horizontalinnermain0"},/area/f13/wasteland)
 "aeA" = (/turf/open/indestructible/ground/outside/sidewalk{dir = 8; icon_state = "outerpavement"},/area/f13/wasteland)
+"aeB" = (/obj/structure/table/wood,/obj/item/katana,/obj/structure/window/reinforced{dir = 4},/turf/open/floor/f13/wood{tag = "icon-housewood2"; icon_state = "housewood2"},/area/f13/building)
+"aeC" = (/obj/item/twohanded/required/kirbyplants/random,/obj/machinery/light{icon_state = "tube"; dir = 8; step_x = 0; step_y = 0},/turf/open/floor/f13/wood{tag = "icon-housewood2"; icon_state = "housewood2"},/area/f13/building)

```

Comment:
You need to remove the empty step_x and step_y variables that got added to your map edit, they exist for pixel movement.
DM places them in if you even mess with the toggle on them.
For future reference use pixel_x and pixel_y for adjustment, or use strongdmm for your map edits as it should clean up. You can open it up in a text editor and ctrl + f for step_x and step_y and just delete the entries.